src/Forms/GridField/GridFieldDeleteAction.php

```diff
@@ -188,6 +204,54 @@ public function handleAction(GridField $gridField, $actionName, $arguments, $dat
         }
     }
 
+    /**
+     *
+     * @param GridField $gridField
+     * @param DataObject $record
+     * @param string $columnName
+     * @return GridField_FormAction|null
+     */
+    private function getRemoveAction($gridField, $record, $columnName)
```

Comment:
Well, this being private means GridFieldGroupDeleteAction can't override this, which was missed during upgrade.